tock/hours/forms.py

```diff
@@ -15,17 +15,20 @@
 from datetime import datetime, timedelta, time
 
 class ReportingPeriodForm(forms.ModelForm):
+    """Form for creating new reporting periods"""
 
     class Meta:
         model = ReportingPeriod
-        fields = ['start_date', 'end_date', 'working_hours', 'message']
+        fields = ['start_date', 'end_date', 'min_working_hours', 'max_working_hours',]
```

Comment:
we never completed linking the `messages` field in the `hours` model, so entries in this form don't actually show up anywhere. doesn't mean we shouldn't fix this at some point, just wanted to clear up confusion for persons creating new reporting periods.